pkg/config/config.go

```diff
@@ -15,7 +15,9 @@ func New() *Config {
 	return &Config{}
 }
 
-func (c *Config) LoadNodeConfig() (string, error) {
+// TODO should this be a function instead of a method?
```

Comment:
Probably

Reply:
I'll leave that to you guys to do -- I don't think the `Config` struct is actually used anywhere (it's empty) and can be removed altogether. Maybe I'm missing something, though

Reply:
Yeah was just a comment from my end agreeing with you. No action on your end required.